app/components/shared/filters_component/filters_component.html.haml

```diff
@@ -0,0 +1,61 @@
+.filters
+  .filters__heading.moj-filter
+    %div{ class: "#{@options[:close_all] ? 'filters__heading-container' : 'moj-filter__header'}" }
+      .filters__section-heading{ class: mobile_modifier('filters__heading') }
+        .moj-filter__heading-title
+          %h2.govuk-heading-s{ class: mobile_modifier('filters__heading-title'), id: "#{@options[:mobile_variant] ? 'mobile-filters-button' : ''}" }
+            = @filters[:title]
+          %span.govuk-body.filters__heading-applied= applied_text
+          %button.govuk-link.govuk-link--no-visited-state.filters--show-mobile-open{ id: 'return-to-results' }
+            = 'Return to results'
+        - if @options[:close_all]
+          %button.govuk-link.govuk-link--no-visited-state.filters__heading-close-all{ id: 'close-all-groups', class: "close-all" }
+            = t('shared.filter_group.close_all_filter_groups')
+
+  .filters__remove
+    - if display_remove_buttons
+      .moj-filter__content
+        .moj-filter__selected
+          .filters__section-heading{ class: 'govuk-!-margin-bottom-2' }
+            .govuk-body{ class: 'govuk-!-margin-bottom-0' }
+              = t('shared.filter_group.current_selected_filters')
+            %button.govuk-link.govuk-link--no-visited-state{ id: 'clear-filters-button' }
+              = t('shared.filter_group.clear_all_filters')
+
+          - @items.each do |group|
```

Comment:
Would it make more sense to name these the same way? E.g `groups.each do |group|` or `items.each do |item|`